app/src/main/res/values-hu/strings.xml

```diff
@@ -273,7 +273,7 @@ Ezzel csak a kiválasztott mappák láthatók, mivel a kizárás és a befoglal
         <b>PROTECT &amp; HIDE PHOTOS, VIDEOS &amp; FILES</b>
         Using pin, pattern or your device’s fingerprint scanner you can protect and hide photos, videos &amp; entire albums. You can protect the app itself or place locks on specific functions of the app. For example, you can’t delete a file without a fingerprint scan, helping to protect your files from accidental deletion.
 
-        <b>Check out the full suite of Simple Tools here:</b>
+        <b>Nézze meg a Simple Tools csomagot itt:</b>
```

Comment:
there are still many untranslated strings here, for example the full rest of the long description and the app title on line 229 :) Thanks